data/json/items/comestibles/mutagen.json

```diff
@@ -21,182 +21,83 @@
     },
     {
         "id" : "mutagen_beast",
+        "copy-from" : "mutagen",
         "type" : "COMESTIBLE",
-        "comestible_type" : "DRINK",
         "name" : "beast mutagen",
         "description" : "A rare substance of uncertain origins.  Causes you to mutate.",
```

Comment:
> I figured at some point they should be individualized so I left them for now.

This actually gives the opposite impression: that they are intentionally the same.